<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import_Export_Data</class>
 <widget class="QWidget" name="Import_Export_Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_dataexport">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>导出数据</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_updatasoft">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>升级程序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sysfileinput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>系统配置文件导入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sysfileoutput">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>系统配置文件导出</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_prompt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>为了保证数据完整传输,请保持U盘15S以上再拔出!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>0</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>U盘数据导入导出</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>超级工程师升级程序会清除所有配置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_dataexport</tabstop>
  <tabstop>btn_updatasoft</tabstop>
  <tabstop>btn_sysfileinput</tabstop>
  <tabstop>btn_sysfileoutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
